zigpy/profiles/zgp.py

```diff
@@ -7,12 +7,12 @@
 
 class DeviceType(t.enum16):
     PROXY = 0x0060
-    PROXY_MIN = 0x0061
+    PROXY_BASIC = 0x0061
```

Comment:
The spec only references the following device types:

<img width="503" alt="image" src="https://github.com/zigpy/zigpy/assets/32534428/09f73b12-4c61-4826-ae40-12adf0ba1109">

Why switch from `minimum` to `basic`? I think it would be better to match the spec's naming as much as possible.

Reply:
I based that off the later v1.1.1 spec (page 87) which seems to have changed from Minimum to Basic.

![image](https://github.com/zigpy/zigpy/assets/14279100/88748052-40fa-40ea-aee2-c83d46c4cff8)

Reply:
Ah, you're right! I had the wrong PDF open.